### PR TITLE
Add using environment var for flow

### PR DESCRIPTION
**Pull request by mattdurham**
#### PR Description

Add environment var check for flow in cmd/agent and remove cmd/agentflow. Specifically not adding to changelog/doc since it's incredibly expermental.

#### PR Checklist

- [NA] CHANGELOG updated
- [NA] Documentation added
- [NA] Tests updated

### CODE REVIEW

File: cmd/agent/main.go [hunk 1]
```diff
@@ -29,7 +29,8 @@ func init() {
 }
 
 func main() {
-	// If Windows is trying to run us as a service, go through that
+
+	// If Windows is trying to runFlow us as a service, go through that
```
Comment (rfratto) [inline cmd/agent/main.go:33]: runFlow us? :)

Comment (rfratto) [reply cmd/agent/main.go:33]: Should we leave a TODO to allow Flow to work as a Windows service?

Comment (mattdurham) [reply cmd/agent/main.go:33]: Silly goland rename and me not double checking. Added a todo but likely not supported til flow graduates to full support.

**Review by rfratto — APPROVED** [review]
Looks fine, though I think it's worth spending time figuring out what the entrypoint for Flow should actually be:

* What are the HTTP endpoints it exposes? 
* Does it use a command lib like cobra? 

cmd/agentflow was just a temporary solution, and while it's fine to bring that directly into cmd/agent it still needs some love and attention before we release.

**Pull request merged**